modules/storage-persistent-storage-volume-encrypt-with-kms-key.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage-aws.adoc
+
+[id="aws-container-persistent-volumes-encrypt_{context}"]
+= Encrypting container persistent volumes on AWS with a KMS key
+
+.Prerequisites
+
+* Underlying infrastructure must contain storage.
+* You must create a customer KMS key on AWS.
+
+.Procedure
+
+Defining a KMS key to encrypt container-persistent volumes on AWS is useful when you have explicit compliance and security guidelines when deploying to AWS.
```

Comment:
There shouldn't be an intro sentences to procedure steps (per the SSG), and this module is missing the intro paragraph. So the perfect solution would be to move this sentence up above the prerequisites to use as the intro paragraph. Can you make that move?

Reply:
Addressed